plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcClient.java

```diff
@@ -123,6 +124,17 @@ PreparedStatement buildSql(ConnectorSession session, Connection connection, Jdbc
 
     boolean isLimitGuaranteed(ConnectorSession session);
 
+    default Optional<String> getTableComment(ResultSet resultSet)
+            throws SQLException
+    {
+        return Optional.ofNullable(emptyToNull(resultSet.getString("REMARKS")));
```

Comment:
`emptyToNull` shouldn't be default implementation IMO. The connectors that don't preserve empty comments (or use some other marker value for missing comments) should have their own impl.